libs/ebpfnetsh/util.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <iostream>
+#include "ebpf_api.h"
+
+std::string
+down_cast_from_wstring(const std::wstring& wide_string);
+
+/**
+ * @brief Parse input string (depicting either interface name, alias, or index) and convert to interface index.
+ *
+ * @param[in] arg Input string.
+ * @param[out] if_index Interface index.
+ *
+ * @retval EBPF_SUCCESS Operation succeeded.
+ * @retval EBPF_INVALID_ARGUMENT Input string could not be converted to interface index.
+ */
+ebpf_result_t
+parse_ifindex(_In_z_ const wchar_t* arg, _Out_ uint32_t* if_index);
```

Comment:
```suggestion
parse_ifindex(_In_z_ const wchar_t* arg, _Out_ uint32_t* ifindex);
```
Make arg name match function name (ifindex not if_index).  Also update line 16.

Reply:
This comment still applies to utilities.h